liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/StartDebugMojoSupport.java

```diff
@@ -289,6 +289,7 @@ protected void copyConfigFiles() throws Exception {
 
         // copy jvm.options to server directory if end-user explicitly set it
         File optionsFile = new File(serverDirectory, "jvm.options");
+        if (optionsFile.exists()) optionsFile.delete();
```

Comment:
This file could exist because it was copied over at line 254 from the configDirectory. If tests are not failing with this change, then we need to add a test to ensure the jvm.options from the configDirectory is correctly located in the serverDirectory after this code runs.